src/Congo.WebApi/Startup.cs

```diff
@@ -57,7 +61,13 @@ public void ConfigureServices(IServiceCollection services)
                 c.SwaggerDoc("v1", new OpenApiInfo { Title = "Congo.WebApi", Version = "v1" });
             });
             services.AddMediatR(typeof(Startup).Assembly);
-            services.AddFluentValidation(fv => fv.RegisterValidatorsFromAssemblyContaining<InsertProductRequestValidator>());
+            services.AddFluentValidation(fv =>
+                fv.RegisterValidatorsFromAssemblyContaining<InsertProductRequestValidator>());
+
+            TypeAdapterConfig.GlobalSettings.NewConfig<CartItem, CartItemResponse>()
```

Comment:
Probably want to move this in to another method or it's own class so that Startup remains clean and minimal